src/BootstrapReport/__init__.py

```diff
@@ -1,7 +1,12 @@
+from .report import ObjectOfInterest
+from .examples._generate_example_data import generate_example_dataset
```

Comment:
Let's not have `_generate_example_data` be user-accessible by default.

Reply:
@jmshapir addressed in c2c4c72